### PR TITLE
Add CAP_SYS_CHROOT for netdata service

### LINKED ISSUE #8359: Not grouping container interfaces on lxc host
##### Bug report summary
Not grouping container interfaces on lxc host 

##### OS / Environment

Proxmox VE 6.1-7
Netdata is running on the virtualization hosts.
```
Linux pve-3 5.3.18-2-pve #1 SMP PVE 5.3.18-2 (Sat, 15 Feb 2020 15:11:52 +0100) x86_64 GNU/Linux
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 10 (buster)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="10"
/etc/os-release:VERSION="10 (buster)"
/etc/os-release:VERSION_CODENAME=buster
/etc/os-release:ID=debian
/etc/os-release:HOME_URL="https://www.debian.org/"
/etc/os-release:SUPPORT_URL="https://www.debian.org/support"
/etc/os-release:BUG_REPORT_URL="https://bugs.debian.org/"

```

##### Netdata version

 netdata v1.20.0-103-nightly

##### Component Name

My guess is it has to do with the cgroup plugin.
#### Scenario 1 Installed from Packagecloud

cat /etc/apt/sources.list.d/netdata_netdata-edge.list
```
# this file was generated by packagecloud.io for
# the repository at https://packagecloud.io/netdata/netdata-edge

deb https://packagecloud.io/netdata/netdata-edge/debian/ buster main
deb-src https://packagecloud.io/netdata/netdata-edge/debian/ buster main
```

dpkg -l netdata
```
||/ Name           Version      Architecture Description
+++-==============-============-============-======================================
ii  netdata        1.20.103      amd64        real-time charts for system monitoring
```
Container info:
Privileged lxc id = 118
Unprivilefed lxc id = 117
image
image

Looking under Network interfaces I can see all interfaces
image

#### Scenario 2 Compiled from source.
git clone https://github.com/netdata/netdata.git --depth=100
./netdata-installer.sh --install /opt --nightly-channel --dont-start-it --disable-cloud

Netdata version
/opt/netdata/usr/sbin/netdata -V
```
netdata v1.20.0-103-nightly
```

With this version It works for unprivileged containers but not for privileged ones.

Container info:
Privileged lxc id = 109
Unprivilefed lxc id = 114
image
image

All other interfaces is available under Network Interfaces
image

#### Logfiles 
[netdata-compiled-error.log](https://github.com/netdata/netdata/files/4311749/netdata-compiled-error.log)
[netdata-deb-error.log](https://github.com/netdata/netdata/files/4311750/netdata-deb-error.log)

### LINKED ISSUE #9368: cgroup errors with lxd 4.2
Hello
on ubuntu bionic and lxd 4.2 snap version netdata does not report cgroup information regarding lxd (no lxd.payload containers on the interface)
I have these errors in the logs:

```
2020-06-18 11:37:51: /opt/netdata/usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : Cannot open file '/proc/net/dev'
2020-06-18 11:37:51: /opt/netdata/usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : cannot read cgroup interface list.
2020-06-18 11:37:51: /opt/netdata/usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : child pid 24263 exited with code 1.
2020-06-18 11:37:51: /opt/netdata/usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : PROCFILE: Cannot open file '/proc/net/dev' (errno 2, No such file or directory)
2020-06-18 11:37:51: /opt/netdata/usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : Cannot open file '/proc/net/dev'
2020-06-18 11:37:51: /opt/netdata/usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : cannot read cgroup interface list.
```

 Cannot open file '/proc/net/dev' is very strange because file exist :

```
ls -l /proc/net/dev
-r--r--r-- 1 root root 0 juin  18 11:54 /proc/net/dev
```

### PR DESCRIPTION
**Pull request by vlvkobal**
##### Summary
Netdata service doesn't have the ability to change namespaces. As a consequence, cgroups plugin can't read information about network interfaces in LXD 4.x containers. We need to add `CAP_SYS_CHROOT` to the `netdata.service` unit file to fix that.

Fixes #8359
Fixes #9368

##### Component Name
daemon

##### Test Plan
Run Netdata on a machine with LXD 4.x.
Check if there are no errors `Cannot switch to mount namespace` in `error.log`.
Check if virtual network interfaces are displayed under the corresponding container section and not in "Network Interfaces".

### CODE REVIEW

**Comment by amoss** [comment]
The change looks simple but I don't know anything about LXD. I'll leave it for @Ferroin and @prologic to review.

**Comment by prologic** [comment]
Having worked in a security company before and having to deal with this kind of thing a fair but, I can say that I'm a bit concerned with this change personally. Adding `CAP_SYS_CHROOT` (_or worse `CAP_SYS_ADMIN`_) to the `netdata` service/binary allows the `netdata` binary process and subprocesses "significant" access beyond potentially their own consternated "space" (_assuming they run in their own namespace/container/etc_).

Basically... If there were to be a 3rd-party (_or even our own, or user contributed code to netdata or our own collectors_) that either a) abused this or b) used ti break out unintentionally well then we have a security problem.

However I will approve as this is a straight forward fix for the aforementioned issues. Just that if I don't mention the very _real_ security concerns here, it _may_ come back to haunt us 👮‍♂️
